Improve prometheus-to-sd usage (#529)
GoogleCloudPlatform/click-to-deploy: k8s/wordpress/README.md

metrics are automatically exported to Stackdriver and visible in
[Stackdriver Metrics Explorer](https://cloud.google.com/monitoring/charts/metrics-explorer).

> The name of each metric starts with the component's name (`mysql` for MySQL and `apache` for wordpress).

Reviewer: ```suggestion
The name of each metric starts with the component's name (`mysql` for MySQL and `apache` for WordPress).
```

> Metrics will be labeled with with `app.kubernetes.io/name` consisting of application's name,

Reviewer: ```suggestion
Metrics are labeled with `app.kubernetes.io/name` consisting of application's name,
```
